lms/djangoapps/teams/views.py

```diff
@@ -136,6 +137,15 @@ def get(self, request, course_id):
         topics = get_alphabetical_topics(course)
         organization_protection_status = user_organization_protection_status(request.user, course_key)
 
+        # We have some frontend logic that needs to know if we have any open, public, or managed teamsets,
+        # and it's easier to just figure that out here when we have them all already
+        has_open_teamset = any(topic for topic in topics if topic['type'] == TeamsetType.open.value)
+        has_public_managed_teamset = any(topic for topic in topics if topic['type'] == TeamsetType.public_managed.value)
+        has_private_managed_teamset = any(
+            topic for topic in topics if topic['type'] == TeamsetType.private_managed.value
```

Comment:
Nit: I think Andrew added a little helper property called `is_private_managed` to the `TeamsetConfig` class the other day.  Small suggestion to add `is_open` and `is_public_managed` properties, too - that would help you condense these lines a bit.

Reply:
Actually, here's a fun little code-golf suggestion:
```python
from collections import Counter
teamset_counts_by_type = Counter([topic['type'] for topic in topics])
# ...
"has_open_teamset": bool(teamset_counts_by_type[TeamsetType.open.value]),
"has_public_managed_teamset": bool(teamset_counts_by_type[TeamsetType.public_managed.value])
"has_managed_teamset": bool(
    teamset_counts_by_type[TeamsetType.public_managed.value] + 
    teamset_counts_by_type[TeamsetType.private_managed.value]
),
# ...
```

Reply:
neat